src/Sarif/Visitors/RemoveOptionalDataVisitor.cs

```diff
@@ -0,0 +1,37 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+using System;
+
+namespace Microsoft.CodeAnalysis.Sarif.Visitors
+{
+    public class RemoveOptionalDataVisitor : SarifRewritingVisitor
```

Comment:
>RemoveOptionalDataVisitor [](start = 17, length = 25)

this implementation is not complete: [https://github.com/Microsoft/sarif-sdk/issues/1411](https://github.com/Microsoft/sarif-sdk/issues/1411)